apps/aecore/include/aec_block_insertion.hrl

```diff
@@ -7,44 +7,45 @@
 %%%
 
 %% Header along with it's hash
--record(node, { header :: aec_headers:header()
-              , hash   :: binary()
-              , type   :: key | micro
-              }).
+-record(chain_node, {
+    header :: aec_headers:header()
+    , hash :: binary()
+    , type :: key | micro
+}).
 
 %% Metadata about the given fork
 %% ForkId is local with regards to the node and for recently gossiped blocks
 %% nodes might disagree about the exact value - fork_id is eventually consistent
 %% across the network
--record(fork_info, { fork_id
-                   , difficulty
-                   , fees
-                   , fraud
-                   }).
+-record(fork_info, {fork_id
+    , difficulty
```

Comment:
I understand what you are coming at, but this is kind of a problem of using automated formatting. One can easily miss fork_id is part of this record. All elements should be aligned in one column.